rust/operator/src/kafka_controller.rs

```diff
@@ -767,26 +763,30 @@ fn build_broker_rolegroup_statefulset(
         .add_volume_mount(LOG_DIRS_VOLUME_NAME, STACKABLE_DATA_DIR)
         .add_volume_mount("config", STACKABLE_CONFIG_DIR)
         .add_volume_mount("tmp", STACKABLE_TMP_DIR)
-        .resources(resources);
-
-    // Use kcat sidecar for probing container status rather than the official Kafka tools, since they incur a lot of
-    // unacceptable perf overhead
-    let mut container_kcat_prober = cb_kcat_prober
-        .image("edenhill/kcat:1.7.0")
-        .command(vec!["sh".to_string()])
-        // Only allow the global load balancing service to send traffic to pods that are members of the quorum
-        // This also acts as a hint to the StatefulSet controller to wait for each pod to enter quorum before taking down the next
+        .startup_probe(Probe {
+            tcp_socket: Some(TCPSocketAction {
+                port: IntOrString::Int(kafka.client_port().into()),
+                ..TCPSocketAction::default()
+            }),
+            failure_threshold: Some(10),
+            period_seconds: Some(2),
+            timeout_seconds: Some(5),
+            ..Probe::default()
+        })
         .readiness_probe(Probe {
             exec: Some(ExecAction {
-                // If the broker is able to get its fellow cluster members then it has at least completed basic registration at some point
-                command: Some(kcat_container_cmd_args(kafka)),
+                command: Some(kafka_container_readiness_probe(
+                    kafka,
+                    &rolegroup_ref.object_name(),
+                    &kafka.namespace().context(ObjectHasNoNamespaceSnafu)?,
+                )),
             }),
-            timeout_seconds: Some(5),
-            period_seconds: Some(2),
+            timeout_seconds: Some(15),
```

Comment:
What do you think of decreasing the timeout to 5s? I would have guessed that this metadata-operation should return rather quick.

Reply:
Not sure if I've resolved the callout correctly, but on my machine that takes 13s:

```
bash-4.4$ date;bin/kafka-broker-api-versions.sh --bootstrap-server simple-kafka-broker-default.kuttl-test-renewing-opossum.svc.cluster.local:9092 | grep -c id;date
Fri Sep  2 07:45:58 UTC 2022
1
Fri Sep  2 07:46:11 UTC 2022
bash-4.4$ 
```

Reply:
yeah i had problems when it was on 5 seconds, so i would even go higher than that?

Reply:
Oh yes awesome great JVM things ^^ Disregard my comment.

Reply:
This should be a huge red flag for merging the PR.